### ISSUE TITLE
Author metadata out of hand

### ISSUE THREAD

**Issue by scopatz** [sympy/sympy-paper#154]
It takes up at least half a page worth of text. This should be shortened to simply the corresponding author.

**Comment by asmeurer** [comment]
The real problem is that we have too many authors :)

I don't know how we could fix this. That's how the template works. We do need to reduce the common institutions into a single footnote (Matthew and Mateusz, Harsh and Shivam, me and you). So that will save us a couple of lines. The paper has crept to more than 20 pages, though (and we don't even have the abstract yet). I think the only fix is to cull some content. I'll open a new issue.

**Comment by certik** [comment]
Is your suggestion to remove all footnotes except one?

**Comment by scopatz** [comment]
Yes, my suggestion is to remove all footnotes except for the corresponding author's.  This is what many other papers do, especially if they have a high author count.

**Comment by asmeurer** [comment]
See https://github.com/sympy/sympy-paper/issues/156
